### PR TITLE
Added a set of MongoDB steps

### PR DESCRIPTION
**Pull request by kenanjasim**
Added a mongodb context to allow expansion within integration tests and added a set of mongo steps which could be used when testing

### CODE REVIEW

File: steps.go [hunk 1]
```diff
@@ -51,11 +53,19 @@ func initSteps(ctx StepAdder) {
 	ctx.Step(`^the following files should exist in the corresponding s3 buckets:$`, TheFollowingFilesShouldExistInS3Buckets)
 	ctx.Step(`I delete the following files from the corresponding s3 buckets:$`, IDeleteFilesFromS3)
 
+	ctx.Step("^I put the following documents in the corresponding collections and databases", IPutDocumentsInMongo)
```
Comment (tigh-latte) [inline steps.go]: Step should end in `:$`

File: steps.go [hunk 2]
```diff
@@ -51,11 +53,19 @@ func initSteps(ctx StepAdder) {
...
+	ctx.Step("^I put the following documents in the corresponding collections", IPutDocumentsInMongoColl)
```
Comment (tigh-latte) [inline steps.go]: Step should end in `:$`

File: steps.go [hunk 3]
```diff
@@ -51,11 +53,19 @@ func initSteps(ctx StepAdder) {
...
+	ctx.Step(`^the following document IDs should exist in the corresponding mongo collections:$`, TheFollowingDocumentsShouldExistInMongoCollections)
+	ctx.Step(`^the following documents should match the following files:$`, TheFollowingDocumentsShouldMatchTheFollowingFiles)
+	ctx.Step(`^the database document should match the following values:$`, TheDocumentShouldMatchTheFollowingValues)
+	ctx.Step(`I drop the following mongo databases :$`, IDropMongoDatabase)
```
Comment (tigh-latte) [inline steps.go]: Step should start with `^`, and there's random whitespace at the end

**Review by tigh-latte — COMMENTED** [review]
Looks good, thanks for the contribution! Just a couple of syntactical things around the step definitions to call out, other than that, looks great.

**Comment by kjasimcq** [comment]
> Looks good, thanks for the contribution! Just a couple of syntactical things around the step definitions to call out, other than that, looks great.

Fixed 😄

**Review by tigh-latte — APPROVED** [review]
LGTM

**Pull request merged**